config.py

```diff
@@ -16,16 +18,34 @@
     env="development",  # this is the active env, by default
     env_switcher="ECOBALYSE_ENV",
     validators=[
+        Validator(
+            "LOG_LEVEL",
+            default="INFO",
+            is_in=(logging.getLevelNamesMapping().keys()),
+        ),
         # Check that the output dir was set
         Validator(
             "OUTPUT_DIR",
             must_exist=True,
             messages={
-                "must_exist_true": "🚨 ERROR: For the export to work properly, you need to specify "
-                + PREFIX
-                + "_{name} env variable. It needs to point to the 'public/data/' directory of https://github.com/MTES-MCT/ecobalyse/ repository. Please, edit your .env file accordingly or add the {name} variable to your `settings.toml` file."
+                "must_exist_true": "🚨 For the export to work properly, you need to specify "
+                "the EB_{name} env variable.\nIt needs to point to the 'public/data/' directory "
+                "of the https://github.com/MTES-MCT/ecobalyse/ repository. \nPlease, edit your .env file "
+                "accordingly."
             },
         ),
+        # The S3 related variables are read from the environment
+        Validator("S3_ENDPOINT", must_exist=True),
+        Validator("S3_REGION", must_exist=True),
+        Validator("S3_ACCESS_KEY_ID", must_exist=True),
+        Validator("S3_SECRET_ACCESS_KEY", must_exist=True),
+        Validator("S3_BUCKET", must_exist=True),
+        Validator("S3_DB_PREFIX", must_exist=True),
```

Comment:
Maybe we should require it only if we are not on the CI? Having to specify « fake » env variables in the CI looks odd